python/lsst/metaserv/schemaToMeta.py

```diff
@@ -189,63 +189,63 @@ def parse(self):
 
-    def _isUnitLine(self, str):
-        return SchemaToMeta._unitLine.search(str) is not None
+    def _isUnitLine(self, theString):
+        return SchemaToMeta._unitLine.search(theString) is not None
 
-    def _isUcdLine(self, str):
-        return SchemaToMeta._ucdLine.search(str) is not None
+    def _isUcdLine(self, theString):
+        return SchemaToMeta._ucdLine.search(theString) is not None
 
-    def _retrUnit(self, str):
-        x = SchemaToMeta._unitLine.search(str)
-        return x.group(1)
+    def _retrUnit(self, theString):
+        result = SchemaToMeta._unitLine.search(theString)
+        return result.group(1)
 
-    def _retrUcd(self, str):
-        x = SchemaToMeta._ucdLine.search(str)
-        return x.group(1)
+    def _retrUcd(self, theString):
+        result = SchemaToMeta._ucdLine.search(theString)
+        return result.group(1)
 
-    def _containsDescrTagStart(self, str):
-        return re.search(r'<descr>', str) is not None
+    def _containsDescrTagStart(self, theString):
+        return '<descr>' in theString
 
-    def _containsDescrTagEnd(self, str):
-        return re.search(r'</descr>', str) is not None
+    def _containsDescrTagEnd(self, theString):
+        return '</descr>' in theString
 
-    def _retrDescr(self, str):
-        x = SchemaToMeta._descrLine.search(str)
-        return x.group(1)
+    def _retrDescr(self, theString):
+        result = SchemaToMeta._descrLine.search(theString)
+        return result.group(1)
 
-    def _retrDescrStart(self, str):
-        x = SchemaToMeta._descrStart.search(str)
-        return x.group(1)
+    def _retrDescrStart(self, theString):
+        result = SchemaToMeta._descrStart.search(theString)
+        return result.group(1)
 
-    def _retrDescrMid(self, str):
-        x = SchemaToMeta._descrMiddle.search(str)
-        return x.group(1)
+    def _retrDescrMid(self, theString):
+        result = SchemaToMeta._descrMiddle.search(theString)
+        return result.group(1)
 
-    def _retrDescrEnd(self, str):
-        if re.search(r'-- </descr>', str):
+    def _retrDescrEnd(self, theString):
+        if re.search(r'-- </descr>', theString):
             return ''
-        x = SchemaToMeta._descrEnd.search(str)
-        return x.group(1)
+        result = SchemaToMeta._descrEnd.search(theString)
+        return result.group(1)
 
-    def _retrIsNotNull(self, str):
-        if re.search(r'NOT NULL', str):
+    def _retrIsNotNull(self, theString):
+        if re.search(r'NOT NULL', theString):
             return '1'
         return '0'
 
-    def _retrType(self, str):
-        arr = str.split()
+    def _retrType(self, theString):
+        arr = theString.split()
         t = arr[1]
         if t == "FLOAT(0)":
             return "FLOAT"
         return t
 
-    def _retrDefaultValue(self, str):
-        if ' DEFAULT ' not in str:
+    def _retrDefaultValue(self, theString):
+        if ' DEFAULT ' not in theString:
```

Comment:
It looks like you could do this more simply by just searching for `r'\s+DEFAULT\s+(\S+)'` and then stripping any trailing comma (although this doesn't handle all possible default values -- for example, the default for a string could include a space).  Perhaps even more simply, you could also use `r'\s+DEFAULT\s+(.+?)[\s,]'` to take anything up to the first space or comma.